### DISCUSSION TITLE
Unable to Access GitHub Account Due to 2FA Issues

### DISCUSSION THREAD

**Discussion by Rahul-jatasra** [community/community#136929] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,
I am writing to request assistance with accessing my GitHub account. Unfortunately, I am unable to log in because I no longer have access to my two-factor authentication (2FA) app. Although I have my GitHub recovery codes, they do not seem to be working, which has left me locked out of my account.
Could you please help me regain access to my account? I kindly request that you disable the 2FA authentication on my account or provide alternative means to verify my identity.
Here are my account details:
GitHub Username: deepak9875
Email Address Associated with GitHub Account: [deepaklakhlan1234@gmail.com](mailto:deepaklakhlan1234@gmail.com)
I appreciate your prompt assistance in resolving this issue.
Thank you.
Sincerely,

**Comment by davevad93** [comment]
Hi @Rahul-jatasra ,

I think you would have a better chance of finding a solution to your problem if you contact support by opening a ticket [here](https://support.github.com/contact/account), select `Sign in, verification, or other account questions` and explain the problem with your other GitHub account.

**Reply by LiteBrite82** [reply]
@Rahul-jatasra, 
@davevad93 is absolutely right, the community does not have the ability to escalate account issues, opening tickets are the only path to resolution for account lockouts. 

I also highly recommend removing any personal identifying information from your discussion (email); its a major security risk.

**Reply by LiteBrite82** [reply]
see also: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)
